ocp_resources/image_digest_mirror_set.py

```diff
@@ -0,0 +1,28 @@
+from ocp_resources.resource import Resource
+
+
+class ImageDigestMirrorSet(Resource):
+    """
+    Create/Manage ImageDigestMirrorSet configuration object. API reference:
+    https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.html
+    Args:
+        imageDigestMirrors (list of dict):
+            e.g. [{source: <str>, mirrors: <list>}, ..., {source: <str>, mirrors: <list>}]
+            - source - the repository that users refer to, e.g. in image pull specifications
+            - mirrors - one or more repositories (str) that may also contain the same images. The order
+                of mirrors in this list is treated as the user’s desired priority
+    """
+
+    api_version = "v1"
```

Comment:
We should get `api_version` automatically from the group

Reply:
Ok, I see what you mean.